### PR TITLE
♻️ refactor: Remove Google's `gemini-pro-vision` model

### PR DESCRIPTION
**Pull request by RubuJam**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [x] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

移除`gemini-pro-vision`模型，并修改相关文件

Remove the `gemini-pro-vision` model and to modify the related files.

#### 📝 补充信息 | Additional Information

收到Google邮件，`gemini-pro-vision`模型将于2024年7月12日被弃用，包括通过API调用该模型

An email was received from Google informing that the `gemini-pro-vision` model will be deprecated on July 12, 2024. This will include the ability to call the model through the API.

### CODE REVIEW

File: src/libs/agent-runtime/google/index.ts [hunk 1]
```diff
@@ -197,19 +197,8 @@ export class LobeGoogleAI implements LobeRuntimeAI {
...
-      // if message are all text message, use vision will return an error:
-      // "[400 Bad Request] Add an image to use models/gemini-pro-vision, or switch your model to a text model."
-      const noNeedVision = messages.every((m) => typeof m.content === 'string');
-
-      // so we need to downgrade to gemini-pro
-      if (noNeedVision) finalModel = 'gemini-pro';
-    }
-
-    return finalModel;
+  private convertModel = (model: string): string => {
```
Comment (arvinxx) [inline src/libs/agent-runtime/google/index.ts]: 不做转换的话就不用这个方法了

File: src/libs/agent-runtime/google/index.test.ts [hunk 2]
```diff
@@ -399,32 +399,9 @@ describe('LobeGoogleAI', () => {
     });
 
     describe('convertModel', () => {
```
Comment (arvinxx) [inline src/libs/agent-runtime/google/index.test.ts]: 测试也可以对应去掉

Comment (RubuJam) [reply src/libs/agent-runtime/google/index.test.ts]: 相关文件均已处理完毕

**Comment by arvinxx** [comment]
等 7.12 那天合并

**Comment by arvinxx** [comment]
话说 PaLM 啥时候下掉？这个 google 有文档不

**Comment by RubuJam** [comment]
这个好像早就被弃了，等我待会把这部分也处理了

This has been abandoned a long time ago, I will deal with this part later

refs：https://ai.google.dev/palm_docs/deprecation
> 话说 PaLM 啥时候下掉？这个 google 有文档不

**Pull request merged**